testData/preferences/margaret.json5

```diff
@@ -0,0 +1,47 @@
+// # Margaret.json5
+//
+// This preference set changes the Windows 10 magnifier preferences settings.
+//
+// ## Testing
+//
+// This changes the following settings from the system:
+//     + FollowFocus: Narrator doesn't follow focus.
+//     + FollowCaret: Narrator now follows text insertion point.
+//     + Invert: Narrator invert the colours.
+//     + Magnification: Magnification is set to 150%.
+//     + MagnificationMode: Magnification mode is set to "Lens mode".
+//     + FollowNarrator: Magnifier follows Narrator cursor.
+//     + FadeToMagIcon: Magnifier collapses into a magnifying glass icon.
+//     + ZoomIncrement: Zoom increment is set to 50.
+//     + UseBitmapSmoothing: Smooth edges of image and text.
+//     + LensHeight: Now the lens height in magnifier lens mode is 50.
+//     + LensWidth: Now the lens width in magnifier lens mode is 50.
+//     + ClassiCDockPosition: The possition of the dock for "Dock mode" is set to left.
+//
+{
+    "flat": {
+        "name": "margaret",
+        "contexts": {
+            "gpii-default": {
+                "name": "Default preferences",
+                "preferences": {
+                    "http://registry.gpii.net/applications/com.microsoft.windows.magnifier": {
+                        "FollowFocus": 0,
+                        "FollowCaret": 1,
+                        "FollowMouse": 1,
+                        "Invert": 1,
+                        "Magnification": 150,
+                        "MagnificationMode": 3,
+                        "FollowNarrator": 0,
+                        "FadeToMagIcon": 1,
+                        "ZoomIncrement": 50,
+                        "UseBitmapSmoothing": 1,
+                        "LensHeight": 50,
+                        "LensWidth": 50,
+                        "ClassiCDockPosition": 0
```

Comment:
Typo here on C? We really need to start to be able to validate these things : P

Reply:
Yes, I agree. It's indeed a typo. Fixing it.